### PR TITLE
Cleanup tooling scripts

### PR DESCRIPTION
**Pull request by ductiletoaster**
Have been wanting to cleanup the tool for a while. Since I did something similar for SSR figured I could do it here. Might actually take some of the learnings I found here to further improve SSR.

### CODE REVIEW

File: packages/pixeloven-core/common/package.json [hunk 1]
```diff
@@ -17,29 +17,32 @@
...
   "devDependencies": {
-    "@pixeloven/tasks": "^6.0.0-beta.22",
+    "@pixeloven-tooling/compiler": "^6.0.0-beta.22",
+    "@pixeloven-tooling/linter": "^6.0.0-beta.22",
+    "@pixeloven-tooling/pretty": "^6.0.0-beta.22",
+    "@pixeloven-tooling/test": "^6.0.0-beta.22",
     "lint-staged": "9.5.0"
   },
   "scripts": {
-    "clean": "pixeloven-tasks clean",
```
Comment (jimmynotjim) [inline packages/pixeloven-core/common/package.json]: Is there no need for clean or document any more?

File: packages/pixeloven-tooling/compiler/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+or using yarn:
+
+```sh
+yarn add --dev @pixeloven-tooling/linter
+```
+
+## Usage
+> Note this package was originally created to encapsulate common dependencies and scripts used during the development of PixelOven. For a full public CLI experience please see our website [pixeloven](https://www.pixeloven.com/).
+
+This simple package wraps tslint and stylelint and a few other dependencies used throughout our workflow. We are often use this package side by side with lint-staged to help improve our code quality.
```
Comment (jimmynotjim) [inline packages/pixeloven-tooling/compiler/README.md]: `We are often` => `We often`

File: packages/pixeloven-tooling/pretty/README.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
+# @pixeloven-tooling/pretty
+
+
+MAKE SURE TO UPDATE ALL PACKAGES WITH THESE NEW PARTS
```
Comment (jimmynotjim) [inline packages/pixeloven-tooling/pretty/README.md]: Is this a note for yourself?

File: packages/pixeloven-tooling/test/README.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
+# @pixeloven-tooling/pretty
```
Comment (jimmynotjim) [inline packages/pixeloven-tooling/test/README.md]: # @pixeloven-tooling/test

File: packages/pixeloven-tooling/test/README.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+
+MAKE SURE TO UPDATE ALL PACKAGES WITH THESE NEW PARTS
+ALSO CREATE A SETUP SCRIPT AND OTHER BUILD TOOLS FOR THE PIPELINE
+ALSO UPDATE ALL THE README AND PACKAGE FILES WITH THE CORRECT HOME PAGE
+TRY TO PIN ALL DEPS INSTEAD OF DOING ^ --- do they still update on publish?
+REMOVE TYPEDOC FOR THIS PROJECT AND MOVE IT TO BE A CLI ADDON -- REMOVE FROM SSR AS WELL
+
+> Pixel Oven tooling linter
```
Comment (jimmynotjim) [inline packages/pixeloven-tooling/test/README.md]: > Pixel Oven tooling test

File: packages/pixeloven-tooling/test/README.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
+# @pixeloven-tooling/pretty
+
+
+MAKE SURE TO UPDATE ALL PACKAGES WITH THESE NEW PARTS
```
Comment (jimmynotjim) [inline packages/pixeloven-tooling/test/README.md]: More todos here that should probably be removed.

File: packages/pixeloven-tooling/test/README.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+> Pixel Oven tooling linter
+
+See our website [pixeloven](https://www.pixeloven.com/) for more information or our [issues](https://github.com/pixeloven/pixeloven/issues) board to report issues associated with this package.
+
+## Install
+
+Using npm:
+
+```sh
+npm install --save-dev @pixeloven-tooling/linter
```
Comment (jimmynotjim) [inline packages/pixeloven-tooling/test/README.md]: npm install --save-dev @pixeloven-tooling/test

File: packages/pixeloven-tooling/test/README.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+```
+
+or using yarn:
+
+```sh
+yarn add --dev @pixeloven-tooling/linter
```
Comment (jimmynotjim) [inline packages/pixeloven-tooling/test/README.md]: yarn add --dev @pixeloven-tooling/test

File: packages/pixeloven-tooling/test/README.md [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+
+## Usage
+> Note this package was originally created to encapsulate common dependencies and scripts used during the development of PixelOven. For a full public CLI experience please see our website [pixeloven](https://www.pixeloven.com/).
+
+This simple package wraps tslint and stylelint and a few other dependencies used throughout our workflow. We are often use this package side by side with lint-staged to help improve our code quality.
+
+Example of usage in `package.json`
+```json
+{
+  "devDependencies": {
```
Comment (jimmynotjim) [inline packages/pixeloven-tooling/test/README.md]: This is all the test config as well

File: packages/pixeloven-core/common/package.json (hunk 1 above)
Comment (ductiletoaster) [reply packages/pixeloven-core/common/package.json]: We do incremental builds now and I never got around to implementing the typedocs stuff. I'm going to support the typedocs stuff for the main CLI for now. Eventually plan on moving it as an addon.

File: packages/pixeloven-tooling/pretty/README.md (hunk 3 above)
Comment (ductiletoaster) [reply packages/pixeloven-tooling/pretty/README.md]: haha opps

File: packages/pixeloven-tooling/test/README.md (hunk 4 above)
Comment (ductiletoaster) [reply packages/pixeloven-tooling/test/README.md]: Don't yell at me

**Pull request merged**